data/i18n/en/en.toml

```diff
@@ -3,6 +3,14 @@
 
 # For "and", see [conjunction_1]
 
+[api_reference_title]
+other = "API reference"
+
+# NOTE: <resource-api-ref> is a placeholder for the text set by 'api-reference-link' shortcode.
+# Please do not localize or modify <resource-api-ref> placeholder.
+[api_reference_text]
+other = "Please refer to the <resource-api-ref> page for more details."
```

Comment:
You could use `%s` and a Hugo `printf` instead (I would).

Reply:
I've updated this in the recent commits as part of the updated approach.